crates/streamer/src/lib.rs

```diff
@@ -7,4 +7,5 @@ pub use config::StreamingConfig;
 pub use streamer::reopenable_file::ReOpenableFile;
 pub use streamer::signal_writer::SignalWriter;
 pub use streamer::{webm_stream, Signal};
-use webm_iterable as _;
+// We need ebml_iterable for TagIterator::emit_master_end_when_eof method
+use ebml_iterable as _;
```

Comment:
I don't see anything imported from this crate. Am I missing something?

Reply:
we depends on webm_iterable which depends on embm_iterable 0.6.2 that does not have emit_master_end_when_eof method. So we manually include latest embm_iterable for that

Reply:
You can update the Cargo.lock with `cargo update -p ebml_iterable` instead. It’s fine to remove the dependency from our Cargo.toml.